app/models/katello/concerns/operatingsystem_extensions.rb

```diff
@@ -3,15 +3,25 @@ module Concerns
     module OperatingsystemExtensions
       extend ActiveSupport::Concern
 
+      REDHAT_ATOMIC_HOST_DISTRO_NAME = "Red Hat Enterprise Linux Atomic Host"
+      REDHAT_ATOMIC_HOST_OS = "RedHat_Enterprise_Linux_Atomic_Host"
```

Comment:
https://github.com/theforeman/foreman/blob/develop/app/models/operatingsystem.rb#L247 -> Makes inferences on family name based on the name of the OS.